logger/httplog/body_logger.go

```diff
@@ -122,7 +123,15 @@ func (b bodyLogger) redactedDump(prefix string, body []byte) string {
 				sb.WriteString("\n")
 			}
 			line = strings.Trim(line, "\r")
-			sb.WriteString(fmt.Sprintf("%s%s", prefix, line))
+			sb.WriteString(prefix)
```

Comment:
It's maybe out of scope for this PR but I'd rather it decompressed the response (based on content-encoding header) and printed it.

If the reason I use debug log is to see the responses then that's what I'd be interested in.